lib/adapter/spec/an_adapter.rb

```diff
@@ -86,11 +86,11 @@
   describe "#key?" do
     it "returns true if key available" do
       adapter.write(key, attributes)
-      adapter.key?(key).should be_true
+      adapter.key?(key).should be true
```

Comment:
Could we swap these to be(false) and be(true)? That would be more consistent with the coding style.